### DISCUSSION TITLE
Windows 10 on ARM BSOD

### DISCUSSION THREAD

**Discussion by giovariot** [utmapp/UTM#6127] [Windows] [answered]
I've tried downloading and installing Windows 10 ARM following the [guide](https://docs.getutm.app/guides/windows/) from UTM documentation. Unfortunately Windows 10 refuses to install giving out a BSOD right after the partitioning phase while still at 0% of "Copying Windows files".

<img width="803" alt="Screenshot 2024-02-10 alle 02 25 20" src="https://github.com/utmapp/UTM/assets/12408904/ff4b202b-3d5c-4a0d-8c9d-89b1d98df588">

When I reboot the system I get a BSOD even before that, right after the rolling windows wheel, instead of getting the darkish blue windows installer background.
After a reboot UTM also gets stuck at "Start boot option" instead of showing the loading bar at the bottom of the screen. I get the same result by resetting, can get Windows to "boot" to the BSOD only turning off the VM and restarting it.

<img width="805" alt="Screenshot 2024-02-10 alle 03 02 49" src="https://github.com/utmapp/UTM/assets/12408904/bf4e8078-f9e3-4c3e-9aa5-42dbede27713">

**Configuration**  
* UTM Version: 4.4.5 (94)
* macOS Version: 14.2.1 (23C71)
* Mac Chip (Intel, M1, ...): M1
* Windows 10 ISO: Windows 10 ARM64 italian from Crystalfetch (filename: 19045.3803.231204-0204.22h2_release_svc_refresh_CLIENTCONSUMER_RET_A64FRE_it-it.iso)

**Debug log**  
The log is not created after closing the VM after the BSOD during the install (the "export debug log" stays greyed out).
After rebooting and therefore not even getting to load the installer I manage to get a debug log. Here it is: 
[debug.log](https://github.com/utmapp/UTM/files/14227543/debug.log)

**Upload VM**  
[Windows 10 ARM.utm.zip](https://github.com/utmapp/UTM/files/14227565/Windows.10.ARM.utm.zip)

**Comment by jackra1n** [comment]
Im having the exact same issue

**Comment by osy** [comment, marked as answer]
See https://docs.getutm.app/guides/windows/#installer-crashes-with-bsod-system_thread_exception_not_handled

"You are using a version of Windows that is too old. The build number should be 21390 or higher."

**Reply by TylerAustin7** [reply]
didn't work for windows 10 22h2 arm64 and I don't wanted to use windows 11 arm

**Comment by changanmoon** [comment]
> didn't work for windows 10 22h2 arm64 and I don't wanted to use windows 11 arm

@bolandnick18 

FYI: Actually, you could run Windows 10 22H2 arm64 in UTM, but you have to emulate it by **disabling the hypervisor**. This is because Microsoft doesn't support any VM hypervisor in Windows 10 22H2 arm64.

Here's what I've configured for the VM on my MacBook Pro 2021 (M1 Pro):
- System: QEMU 7.2 ARM Virtual Machine (alias of virt-7.2) (virt）
- CPU: max
- 8 GB RAM, 4 cores
- Force multicore enabled
- UEFI and RNG device enabled, TPM 2.0 device and hypervisor disabled

And here's the screenshot (Even though the interface language is Traditional Chinese, you can see that the Windows version is 22H2):
<img width="1140" alt="screenshot" src="https://github.com/utmapp/UTM/assets/46277006/e3d57603-8895-4445-a0ab-55f4c20ac28e">

However, even though I've installed Windows 10 22H2 arm64 successfully in VM, it took me a long time to go through OOBE. 

What's more, its performance is so laggy (even poor than emulating Windows 10 22H2 x64) that I couldn't run SPICE guest tools installer at all.

Therefore, for better experience on Apple Silicon Macs, it is not recommended to run (emulate) Windows 10 22H2 arm64, instead, virtualizing Windows 11 arm64, or Windows 10 arm64 Insider (21390 and above).

**Reply by fbuffa2003** [reply]
This works for me on M2. Is very slow but at least is working. Windows 11 works perfect.

**Comment by github11122292029** [comment]
yeah, i agree but for some reason its fast.

**Comment by github11122292029** [comment]
my fan will get loud

**Comment by github11122292029** [comment]
Maybe try 1709 or 1809 without tpm or hypervisor

**Comment by diego351** [comment]
[Windows 10 21390 ARM](https://github.com/WhatTheBlock/WindowsSimplify/discussions/64#discussioncomment-5586168)

VM, fast mode. Remember to turn off TPM in the VM settings

**Comment by JustSomeGitHubUser693** [comment]
I get the error, but I couldn't find a 22H2 version
